core/logging_api.php

```diff
@@ -125,11 +126,18 @@ function log_event( $p_level, $p_msg ) {
 
 	$t_php_event = $t_now . ' ' . $t_level . ' ' . $t_msg;
 
+	if( isset( $t_modifiers ) && !( is_writable ( $t_modifiers ) ) && $t_call_once ){
+		trigger_error( "Warning : $t_modifiers is not a writable file", E_USER_WARNING );
```

Comment:
Localize the error message by adding it to strings_english.txt and referencing it as a localized string.